worklets/Overview.bs

```diff
@@ -185,14 +185,13 @@ When a user agent is to <dfn>create a WorkletGlobalScope</dfn>, for a given |wor
 
         1. Let |resolvedModuleURL| be |entry|'s key.
 
-        2. Let |script| be the result of <a>fetch a module script tree</a> given
-            |resolvedModuleURL|, "anonymous" for the <a>CORS setting attribute</a>, and
-            |settingsObject|.
+        2. <a>Fetch a module worklet script tree</a> given |resolvedModuleURL|, |outsideSettings|,
```

Comment:
Yeah this was wrong. This only should be loading the top level requests.

This section is used for if a user-agent wants to create a new WorkletGlobalScope in addition to the ones it already has. (E.g. it may have killed one from the pool and needs to restart or something).

When instantiating the new WorkletGlobalScope it has to load all the previous top level modules loaded into it.

Added a "top level module url list" to the worklet which the fetch hook now populates. This reads off that to load it's code.